src/header.ts

```diff
@@ -179,9 +184,21 @@ export class AppHeader {
    */
   constructor(private parent: HTMLElement, options: IAppHeaderOptions = {}) {
     mixin(this.options, options);
+    this.addEUCookieDisclaimer();
```

Comment:
please make it an optional feature that can be en/disabled using an option.

Reply:
I decided for opt-in, so we have to set the flag for every application.